tools/accuracy_checker/accuracy_checker/metrics/regression.py

```diff
@@ -523,6 +523,8 @@ def calculate_distance(x_coords, y_coords, selected_points):
 
 
 def mae_differ(annotation_val, prediction_val):
+    if (annotation_val.shape[0] < prediction_val.shape[0]):
```

Comment:
here can be not only arrays, but also just float values, possibly you should handle this case